Dockerfile-parallel

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.4.2
+
+COPY Gemfile Gemfile
+COPY Gemfile.lock Gemfile.lock
+
+RUN bundle install
+
+COPY features /features
+
+ENTRYPOINT ["bundle", "exec", "parallel_cucumber", "features/"]
```

Comment:
Why not include the `--strict` option? It ensures that the tests fail if there are missing or undefined steps (cucumber just keeps going by default). Or does `parallel_cucumber` not support it?

Reply:
That was accidental. I can put it back. However, reading about the strict mode, when enabled it doesn't allow us to have pending tests. That's something @chrisclayson actually needed and implemented in https://github.com/alphagov/verify-acceptance-tests/pull/43 (we deleted it). 

IMHO, pending/incomplete test is not same as a failing one. Especially in TDD or feature-flagged context...

Reply:
I think we added it back in the day because cucumber would keep running if there were tests using undefined steps (e.g. when someone forgot to add a new step). What are pending tests and why do we need them?
No big deal either way, AFAIK `strict` mode just makes cucumber fail faster if it encounters an issue.

Reply:
Ah, I see, I didn't have that context.

In our scenario, we wrote `pending` tests before our new feature was fully implemented and enabled in all environments. We wanted the tests there before the feature itself and the strict mode was failing the whole test suite. Chris then implemented a custom annotation to ignore the specific tests (which could had been achieved by removing the strict mode I guess)...

I'm not bothered either way, it's good to see both views... I'll put the strict mode back in unless anyone else thinks I shouldn't...

Reply:
done